board/views.py

```diff
@@ -183,27 +74,21 @@ def __init__(self):
 
 
 def main(request):
-    params_post = {}
+    params = {}
     initials = {}
 
     data = request.GET
 
     if 'platforms' in data:
-        params_post['filter[platforms][eq]'] = '(' + ','.join(data.getlist('platforms')) + ')'
+        params['filter[platforms][eq]'] = '(' + ','.join(data.getlist('platforms')) + ')'
```

Comment:
The view should know nothing about how igdb api is structured. Filters should be passes ad arguments, smth like
```python
igdb_wrapper.get_games(platforms=['PS5'], genres=['Action'])
```